internal/testcontext/compile_norace.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build !race
+
+package testcontext
+
+const raceEnabled = false
```

Comment:
How does this work: two constants with the same name and different values in the same package?

Reply:
They have different build tags.

Reply:
Just to clarify, the build tags in these files are exclusive. So one or the other will always be included, but never both at the same time. Hence it's fine to have the same name.